core/lib/web3_decl/src/namespaces/zks.rs

```diff
@@ -46,7 +46,10 @@ pub trait ZksNamespace {
     async fn get_bridgehub_contract(&self) -> RpcResult<Option<Address>>;
 
     #[method(name = "getMainContract")]
-    async fn get_main_contract(&self) -> RpcResult<Address>;
+    async fn get_main_l1_contract(&self) -> RpcResult<Address>;
+
+    #[method(name = "getL2Multicall3")]
+    async fn get_l2_multicall3(&self) -> RpcResult<Option<Address>>;
```

Comment:
Why do we need this as an RPC method? Any new method is a PITA to live with. Can we live without it?

Reply:
We need it for getting l2 multical address from the gateway node. We can't predict this value

Reply:
I agree, that we can improve, this particular part. 
But probably we need to slightly restructure the initial set of methods and make the one method with all necessary contracts, that will deprecate all this small methods. 

Let's plan it and implement as the next step

Reply:
PLA-1105